Gemfile

```diff
@@ -14,7 +14,7 @@ gem 'api-pagination'
 gem 'attachinary'
 gem 'autoprefixer-rails'
 gem 'bootstrap-sass', '~> 3.3.5'
-gem 'central-support', github: 'Codeminer42/cm42-central-support', require: 'central/support'
+gem 'central-support', github: 'Codeminer42/cm42-central-support', require: 'central/support', branch: 'fix/csv-export-attachments'
```

Comment:
I just merged the PR from the `cm42-central-support` repository, we can remove the `branch` option now 😄